ceph.spec.in

```diff
@@ -468,6 +470,9 @@ BuildRequires:  libnuma-devel
 %if 0%{?rhel} >= 8
 BuildRequires:  /usr/bin/pathfix.py
 %endif
+%if 0%{with mgr_dashboard_frontend_support_multi_language}
+BuildRequires:  npm
```

Comment:
Why is system NPM required specifically for multi-language support? I don't get why i18n is making it required. I mean, I think it'd be better to use distro-packages as much as possible, but in that case NPM is required for building the Dashboard UI:
```suggestion
%if 0%{with mgr_dashboard_frontend_build}
BuildRequires:  npm
```

Reply:
> Why is system NPM required specifically for multi-language support? I don't get why i18n is making it required. I mean, I think it'd be better to use distro-packages as much as possible, but in that case NPM is required for building the Dashboard UI:

We need to use npm to build dashboard UI when we execute command about `rpmbuild -ba ceph.spec`, You mean we should to modify the make-dist script to support multi-language to replace build in ceph.spec？see: https://github.com/ceph/ceph/blob/main/make-dist#L143

Reply:
I mean that if `npm` is required for multi-language, it's also required for any Dashboard front-end build, as everything in the UI is built with npm.

Reply:
I completely agree with your statement.

Reply:
Cool. Then I'll proceed to give my approval! Thank you @TomNewChao !